src/openforms/forms/tests/test_api_form_plugin_options.py

```diff
@@ -151,3 +151,89 @@ def test_payment_backend_options(self):
                 self.assertEqual(
                     json["invalidParams"][0]["name"], "paymentBackendOptions.email"
                 )
+
+    def test_overwrite_only_registration_email_subject_templates(self):
+        form = FormFactory.create(
+            registration_backend="email",
+            registration_backend_options={"to_emails": ["test@test.nl"]},
+        )
+
+        url = reverse("api:form-detail", kwargs={"uuid_or_slug": form.uuid})
+
+        response = self.client.patch(
+            url,
+            data={
+                "registration_email_subject": "Custom subject",
+            },
+        )
+
+        self.assertEqual(response.status_code, 200)
+
+        data = response.json()
+
+        self.assertEqual(data["registrationEmailSubject"], "Custom subject")
+        self.assertEqual(data["registrationEmailPaymentSubject"], "")
+        self.assertEqual(
+            data["registrationEmailContentHtml"],
+            "",
+        )
+        self.assertEqual(
+            data["registrationEmailContentText"],
+            "",
+        )
```

Comment:
We haven't set the asserted values explicitly to `""`, that's a behaviour from a Factory/model, right?
Then it's saver to check that they are the same to whatever the factory built.

Reply:
Yes indeed, it's the behaviour of the model, since these attributes are by default empty! I will change the assertion.